Pedigree.py

```diff
@@ -896,6 +933,43 @@ def readInSequence(self, fileName, startsnp=None, stopsnp = None):
             ind.reads[e][:] = reads
             e = 1-e
 
+    def readInAAP(self, fileName):
+        """function for reading alternative allele probability input
+
+        :param fileName: The file path
+        :type fileName: str
+        """        
+        data_list = MultiThreadIO.readLines(fileName, startsnp=None, stopsnp=None, dtype = np.int64)
+
+        # flag of whether adding a default alternative allele probability
+        default_aap = False
+        MainMetaFounder = self.MainMetaFounder
+        if MainMetaFounder:
+            default_aap = True
+
+        for value in data_list:
+            idx, data = value
```

Comment:
This function reads in the user-inputted alternative allele probabilities for each metafounder. The first column is the defined metafounder (MF_x), and the remaining columns include the alternative allele probability at each locus. So, here idx is equal to MF_x. If so, would it be better to change `idx` to `mfx` to avoid confusion with the user-inputted pedigree id?

Reply:
Good idea, I will change it in the next commit.